util/src/sha3.rs

```diff
@@ -68,6 +68,24 @@ impl<T> Hashable for T where T: AsRef<[u8]> {
 	}
 }
 
+impl<T> Hashable for [T] where T: Hashable {
+	fn sha3(&self) -> H256 {
```

Comment:
and btw, just fyi, xoring is the preferred way to combine multiple hashes (that is proven to be as good as the individual hashes are themselves), so that's quite natural to expect this implementation when hashing list of hashes